packages/client/src/containers/DropdownGroup/DropdownGroup.Container.js

```diff
@@ -2,11 +2,34 @@ import React from 'react';
 import ViewDropdown from '../../components/ViewDropDown/ViewDropdown.component';
 import './DropdownGroup.Style.css';
 
-export const DropdownGroup = () => {
+export const DropdownGroup = (productId) => {
   const Colors = ['Black', 'White', 'Blue', 'Half-white'];
   const Sizes = ['Small', 'Medium', 'Large', 'XL', 'XXL'];
   const Quantities = [1, 2, 3, 4, 5, 6, 7, 8, 9];
```

Comment:
Why do these have capital names?

Reply:
I will change them

Reply:
Hej @miltonhultgren it doesn't have any meaning to write.. but I did usual, and we made changes... thank you ..